docs/WALKTHROUGH.md

```diff
@@ -141,7 +141,7 @@ submitted.
     swap id, amount, and current status as a progress bar.
 
     Swaps require on-chain transactions, so you will need to wait for confirmations before
-    they will complete successfully. For more detailed information about the swap process,
+    they will be completed successfully. For more detailed information about the swap process,
```

Comment:
imo saying "the swap will complete" makes sense. So I think it was fine before? but i guess also makes sense to say it "will be completed" but I think both work tbh

Reply:
The section can probably be rewritten given that certain swaps can be preregistered, obviating the need for confirmations.